sdk/servicebus/service-bus/src/receiver.ts

```diff
@@ -19,10 +19,10 @@ import {
   throwErrorIfConnectionClosed,
   throwTypeErrorIfParameterMissing,
   throwTypeErrorIfParameterNotLong,
-  throwTypeErrorIfParameterNotLongArray,
-  getErrorMessageNotSupportedInReceiveAndDeleteMode
+  throwTypeErrorIfParameterNotLongArray
 } from "./util/errors";
 
+import { getErrorMessageNotSupportedInReceiveAndDeleteMode } from "./util/errors";
```

Comment:
The changes in this file need to be reverted

Reply:
To? It's being imported from errors.ts now correct?

Reply:
Ok, looks like you meant grouping into 1 import (wonder why tslint didn't do that :) )
Updated.